# HG changeset patch
# User Rik <rik@octave.org>
# Date 1513033183 28800
#      Mon Dec 11 14:59:43 2017 -0800
# Branch stable
# Node ID 62add1b6910047e1f05ae3f3771a2fcdedad63ea
# Parent  a7dfb685d261e6cae088e84a4d6c2f35e59360dd
Add polarplot() to the list of unimplemented functions (bug #52643).

* __unimplemented__.m: Add special case for polarplot() which gives a message
pointing to polar() as the current workaround.

diff --git a/scripts/help/__unimplemented__.m b/scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m
+++ b/scripts/help/__unimplemented__.m
@@ -84,16 +84,19 @@ function txt = __unimplemented__ (fcn)
              'Octave uses the file ".octaverc" instead.'];
 
     case {"ode113", "ode15i", "ode15s", "ode23s", "ode23t", "ode23tb"}
       txt = ["Octave provides lsode and ode45 for solving differential equations. ", ...
              "For more information try @code{help lsode}, @code{help ode45}.  ", ...
              "Matlab-compatible ODE functions are provided by the odepkg ", ...
              "package.  See @url{http://octave.sourceforge.net/odepkg/}."];
 
+    case "polarplot"
+      txt = ["polarplot is not implemented.  Consider using polar."];
+
     case "startup"
       txt = ["startup is not implemented.  ", ...
              'Octave uses the file ".octaverc" instead.'];
 
     case "quad2d"
       txt = ["quad2d is not implemented.  Consider using dblquad."];
 
     case {"xlsread", "xlsfinfo", "xlswrite", "wk1read", "wk1finfo", "wk1write"}
